src/components/LoginModal/index.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  Modal,
+  ModalVariant,
+  Button,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { routerRedux } from 'dva/router';
+import { connect } from 'dva';
+import LoginForm from '@/components/LoginForm';
+
+@connect(auth => ({
+  auth: auth.auth,
+}))
+class LoginModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalOpen: false,
+      modalView: 0,
```

Comment:
Can true and false be used for `modalView` state tracking?

Reply:
fixed!